OWNERS

```diff
@@ -1,16 +1,72 @@
-approvers:
-- enxebre
-- csrwng
-- sjenning
-- davidvossel
-- muraee
-- bryan-cox
-- jparrill
-options: {}
-reviewers:
-- enxebre
-- csrwng
-- sjenning
-- hasueki
-- rtheis
+filters:
+  ".*":
+    approvers:
+    - core-approvers
+    reviewers:
+    - core-reviewers
+    emeritus-approvers:
+    - davidvossel
+  # KubeVirt specific OWNERS
+  "(cmd|control-plane-operator|hypershift-operator|product-cli)(/.*)?/.*kubevirt.*":
+    approvers:
+    - kubevirt-approvers
+    reviewers:
+    - kubevirt-reviewers
+  "test/integration":
+    approvers:
+    - tech-leads
+    reviewers:
+    - core-reviewers
+  # Labels for areas sorted alphabetically by directory.
+  # Note: control-plane-pki-operator also has approvers defined.
+  "api":
+    labels:
+    - area/api
+  "availability-prober":
+    labels:
+    - area/control-plane-operator
+  "cmd":
+    labels:
+    - area/cli
+  "control-plane-operator":
+    labels:
+    - area/control-plane-operator
+  "control-plane-pki-operator":
+    labels:
+    - area/control-plane-pki-operator
+    approvers:
```

Comment:
Why does this section have tech leads specifically called out?

Reply:
it's how it is now, the OWNERS file under the `control-plane-pki-operator` has @sjenning , @csrwng and @enxebre as approvers. So they'd be preferred when suggested as approvers.